### DISCUSSION TITLE
【灌水】我可没想过带节奏

### DISCUSSION THREAD

**Discussion by ehan0828** [share121/inter-knot#5120] [灌水]
![af64e899e640a2f14c7352695bfe0e1](https://github.com/user-attachments/assets/fdeb004d-e9a5-4e47-b476-c725600ce6d9)

前段时间，我发了个有关亚食人暴雷的帖子，吐槽了亚食人顺便发表了一下我对她的不满，然后我的帖子就被热心网友们说是带节奏举报了，我也变成“搬史仙人”了<img src="https://bbs-static.miyoushe.com/static/2024/07/31/1a2616d760a6f334a4a9abca46267af3_7281603850910753959.png" alt="三月七 我" width="75" >
在此，我发表我的声明：我只是吐槽了亚食人顺便发表了一下我对她的不满，没有想过学那帮“搬史仙人”带节奏。我也不是什么所谓的“搬史仙人”<img src="https://bbs-static.miyoushe.com/static/2024/07/31/bfaa94e859e32501b1ffcf662fcef4be_6155242370256719352.png" alt="机巧鸟 尖叫" width="75" >
话已至此，如果还会有人觉得我是“搬史仙人”然后把这条帖子举报、来这条帖子下面不依不饶的认为我是带节奏的“搬史仙人”或者是认为我在对号入座是被害妄想症急了的话<img src="https://bbs-static.miyoushe.com/static/2024/07/31/33f69e7df0fb9d06592379f11d7ba6ba_1797154502027561525.png" alt="云璃 威慑" width="75" >
那我建议你他宝贝的去多听几遍“一颗方糖悬滞的时间”好好让自己这充满戾气的心和容易上头的精神状态冷静下来吧，宝贝的绝了<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >

**Comment by ehan0828** [comment]
另外，如果有求药使想要加入药王秘传的话，跟我私聊，现在加入和兄弟姐妹们相处还有福利哟<img src="https://bbs-static.miyoushe.com/static/2024/07/31/480ca63155fad40e5ed3b2d25fd7d86d_6740019137875438596.png" alt="怀炎 吃糖" width="75" >

**Reply by fzdqz** [reply]
丰饶孽物？举报了

**Comment by JiuXiat** [comment]
对于你说的问题，我总结回答你一下：
对于你说的问题，我想说的是你那个帖子已经带有节奏向性帖子，这种节奏像性在深网里面是严命禁止的，所以是会被举报

**Comment by cpu644** [comment]
再普通的帖子沾上点负面性都会导致理性讨论演变成双方互撕，除非是个没有争议的事实（eg.不想开学）。不建议这么发。
